.github/workflows/release-dotnet-provider.yml

```diff
@@ -1,11 +1,7 @@
 on:
   push:
-    tags:
-      - v*.*.*
-    paths-ignore:
-      - "CHANGELOG.md"
-      - "CHANGELOG_PENDING.md"
-      - "README.md"
+    branches: [ main ]
+    paths: [ CHANGELOG.md ]
```

Comment:
I think this makes sense, whenever changes to CHANGELOG.md get merged to main we'll create a new release.  

Would it also make sense to have the GitHub action tag the release automatically?  With the old process we had a guarantee that the release is tagged properly, as we'd only kick off the action when a new tag gets pushed, but there's no such guarantee anymore.

Reply:
When goreleaser publishes the release to GitHub it makes a matching tag.